### ISSUE TITLE
 EURO DEMO severe\fatal bicycle\scooter involvers (group_bar+line)

### ISSUE THREAD

**Issue by yuvalbar23** [hasadna/anyway#1555]
**Bar chart**
JSON:
{
"name": "severe_bicycle_scooter_bar",
"data": {
"items":
[
    {
        "accident_year": 2008,
        "type": "אופניים",
        "involvers": 7
    },
    {
        "accident_year": 2009,
        "type": "אופניים",
        "involvers": 13
    },
    {
        "accident_year": 2010,
        "type": "אופניים",
        "involvers": 9
    },
    {
        "accident_year": 2011,
        "type": "אופניים",
        "involvers": 9
    },
    {
        "accident_year": 2012,
        "type": "אופניים",
        "involvers": 23
    },
    {
        "accident_year": 2013,
        "type": "אופניים",
        "involvers": 16
    },
    {
        "accident_year": 2014,
        "type": "אופניים",
        "involvers": 14
    },
    {
        "accident_year": 2014,
        "type": "אופניים חשמליים",
        "involvers": 4
    },
    {
        "accident_year": 2014,
        "type": "קורקינט חשמלי",
        "involvers": 2
    },
    {
        "accident_year": 2015,
        "type": "אופניים",
        "involvers": 11
    },
    {
        "accident_year": 2015,
        "type": "אופניים חשמליים",
        "involvers": 13
    },
    {
        "accident_year": 2016,
        "type": "אופניים",
        "involvers": 26
    },
    {
        "accident_year": 2016,
        "type": "אופניים חשמליים",
        "involvers": 19
    },
    {
        "accident_year": 2016,
        "type": "קורקינט חשמלי",
        "involvers": 4
    },
    {
        "accident_year": 2017,
        "type": "אופניים",
        "involvers": 15
    },
    {
        "accident_year": 2017,
        "type": "אופניים חשמליים",
        "involvers": 19
    },
    {
        "accident_year": 2017,
        "type": "קורקינט חשמלי",
        "involvers": 2
    },
    {
        "accident_year": 2018,
        "type": "אופניים",
        "involvers": 14
    },
    {
        "accident_year": 2018,
        "type": "אופניים חשמליים",
        "involvers": 22
    },
    {
        "accident_year": 2018,
        "type": "קורקינט חשמלי",
        "involvers": 10
    },
    {
        "accident_year": 2019,
        "type": "אופניים",
        "involvers": 15
    },
    {
        "accident_year": 2019,
        "type": "אופניים חשמליים",
        "involvers": 21
    },
    {
        "accident_year": 2019,
        "type": "קורקינט חשמלי",
        "involvers": 19
    }
]
}
}

**Line chart**

JSON:
{
"name": "severe_bicycle_scooter_line",
"data": {
"items":
[
    {
        "accident_year": 2008,
        "involvers": 7
    },
    {
        "accident_year": 2009,
        "involvers": 13
    },
    {
        "accident_year": 2010,
        "involvers": 9
    },
    {
        "accident_year": 2011,
        "involvers": 9
    },
    {
        "accident_year": 2012,
        "involvers": 23
    },
    {
        "accident_year": 2013,
        "involvers": 16
    },
    {
        "accident_year": 2014,
        "involvers": 20
    },
    {
        "accident_year": 2015,
        "involvers": 24
    },
    {
        "accident_year": 2016,
        "involvers": 49
    },
    {
        "accident_year": 2017,
        "involvers": 36
    },
    {
        "accident_year": 2018,
        "involvers": 46
    },
    {
        "accident_year": 2019,
        "involvers": 55
    }
]
}
}

![image](https://user-images.githubusercontent.com/63560246/96687846-1c3ae080-1389-11eb-9a2f-caff1c9b35cf.png)

**Comment by galraij** [comment]
to complex to show, I think it would be better to show stacked bar chart, it will also serve the idea of showing the combined number which is showed here by the regression line.
![stacked-bar-example-1](https://user-images.githubusercontent.com/3368312/96691192-41c9e900-138d-11eb-8c86-9f0ff4f0272a.png)

**Comment by yuvalbar23** [comment]
But then you can't notice the trend for each group.
for example, in my chart, you can easily notice a decrease in regular bicycles and an increase in e-bikes and scooters. 
with stacked bar chart it will be possible to easily notice a trend for only the bottom group and for all groups together.

I'll add stacked bar chart so you could choose which one you want.

**Comment by atalyaalon** [comment]
I agree with @yuvalbar23  - it should either be 3 different graphs - or non-stack graph with colors the way he created.
@galraij @MichalOren  any thoughts?

**Comment by MichalOren** [comment]
I support the idea of each type individually. And if to unite something then electric vehicles (scooter and electric bike). regular bikes separately.

**Comment by atalyaalon** [comment]
Thank you @MichalOren!
@yuvalbar23 any thoughts?

**Comment by yuvalbar23** [comment]
LMK what you think

![Line](https://user-images.githubusercontent.com/63560246/99564148-c2065d00-29d2-11eb-9df5-0be3609c7ef3.png)
![stacked bar](https://user-images.githubusercontent.com/63560246/99566342-6f7a7000-29d5-11eb-9059-fbe68c9ff6f6.png)
![category bar](https://user-images.githubusercontent.com/63560246/99566486-a5b7ef80-29d5-11eb-8b45-e81d0c49d531.png)

**Comment by atalyaalon** [comment]
@yuvalbar23 the 3 look very good - I suggest adding the 3 graphs as 3 different widgets.
@galraij @MichalOren  any thoughts?

**Comment by atalyaalon** [comment]
@MichalOren can you open Widgets for the above 3 graphs?

**Comment by yuvalbar23** [comment]
[SQL query for line chart](https://app.redash.io/hasadna/queries/616804/source?p_city=5000&p_year_lower_bound=2008&p_year_upper_bound=2020)

[SQL query for stacked bar chart](https://app.redash.io/hasadna/queries/616592/source?p_city=5000&p_year_lower_bound=2008&p_year_upper_bound=2020)

[SQL query for e-bikes bar chart](https://app.redash.io/hasadna/queries/616803/source?p_city=5000&p_year_lower_bound=2008&p_year_upper_bound=2020)
